models/Type.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Type Model
+ *
+ * Represents categories/death types for projects (classifies why projects were abandoned/buried).
+ *
+ * @note Could add `iconUrl` field if that's something frontend wants.
+ */
+
+module.exports = (sequelize, Sequelize) => {
+  const { DataTypes } = Sequelize;
+
+  const Type = sequelize.define(
+    "Type",
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        autoIncrement: true,
+        primaryKey: true,
+      },
+      name: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        unique: true,
+        comment: "Label for the death type/category",
+      },
+    },
+    {
+      timestamps: true,
+      tableName: "Types",
+    }
+  );
+
+  Type.associate = (models) => {
+    Type.belongsToMany(models.Project, {
+      through: "ProjectTags",
+      foreignKey: "typeId",
+      otherKey: "projectId",
+      as: "projects",
+      onDelete: "CASCADE",
```

Comment:
If were going to be using these for sorting, filtering and the likes I would prefer restricted deletions as we can't have project without types associated with them.
```suggestion
    Type.belongsToMany(models.Project, {
      through: "ProjectTags",
      foreignKey: "typeId",
      otherKey: "projectId",
      as: "projects",
      onDelete: "RESTRICT", 
```